mobile/src/TableCell.js

```diff
@@ -12,11 +12,17 @@ export default class TableCell extends Component {
   }
 
   renderCell = item => {
-    const { primaryColor, currentUser } = this.props
+    const { primaryColor, currentUser, answerButtonPrompt } = this.props
     const voteCount = item.questionId
       ? this.props.votesByAnswer[item.id] || 0
       : this.props.votesByQuestion[item.id] || 0
 
+    let buttonPrompt = this.props.commentsTotal === 1
+      ? t('one_answer')
+      : t('answers_count', { count: this.props.commentsTotal })
+    if (answerButtonPrompt) buttonPrompt = `${this.props.commentsTotal} ${answerButtonPrompt}${this.props.commmentsTotal === 1 ? "": "s"}`
```

Comment:
Can't just add an `s` to pluralize arbitrary text (esp w/ non-English)

Maybe simpler to ask for a plural prompt, and format like `Prompts: 0`, `Prompts: 1`, `Prompts: 2` etc.